README.rst

```diff
@@ -151,3 +151,18 @@ with the links per entity. Example file:
 
 For available icon names, see the `Font Awesome gallery with free icons <https://fontawesome.com/icons?d=gallery&m=free>`_.
 
+
+--------
+Settings
+--------
+
+You can run ``pipenv run python -m kube_resource_report --help`` to find out information.
```

Comment:
I would use `docker run ...  --help` here as this is easier for people (no need to install Pipenv)

Reply:
Done. I also do not know if this approach is "correct". I was also thinking about adding these guys as a `click.option`. But, it can be done in another step I think.